tcmur_cmd_handler.c

```diff
@@ -553,6 +553,8 @@ int tcmur_cmd_handler(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 		/* Try to passthrough the default cmds */
 		if (rhandler->handle_cmd)
 			ret = handle_passthrough(dev, cmd);
+		else
+			tcmu_warn("unknown command %x\n", cdb[0]);
```

Comment:
Since you added another null handle_cmd check to tcmur_cmd_passthrough_handler, I think you can just kill the above check and add the warning message to tcmur_cmd_passthrough_handler's check.